packages/front-end/components/Experiment/Results.tsx

```diff
@@ -107,7 +110,10 @@ const Results: FC<{
       weight: phaseObj?.variationWeights?.[i] || 0,
     };
   });
-
+  const sortedVariations = putBaselineVariationFirst(
```

Comment:
I don't love the reordering here. Seems like we might want to preserve the original order and let specific components that need reordering to handle it as needed. We can chat about this though offline

Reply:
This is currently the plan and the new implementation, although this branch is a stub until you integrate it with your new FE components, reordering as needed